tests/virtualization/vagrant/boxes/tumbleweed.pm

```diff
@@ -23,11 +23,16 @@ use testapi;
 use utils;
 use vagrant;
 use File::Basename;
+use Utils::Architectures 'is_x86_64';
 
 
 sub run() {
+    my $is_virtualbox_applicable = 0;
+    if (is_x86_64) {
+        $is_virtualbox_applicable = 1;
+    }
```

Comment:
could you make that just `my $is_virtualbox_applicable = is_x86_64();`

Reply:
Much better, indeed. :)
Done.